io-engine/src/core/reactor.rs

```diff
@@ -288,15 +288,18 @@ impl Reactor {
             incoming: crossbeam::queue::SegQueue::new(),
             lcore: core,
             flags: Cell::new(ReactorState::Init),
+            tid: Cell::new(0),
```

Comment:
does it need to be a Cell?

Reply:
We need to set the value here in static RO context, as reactor instances are created before we know what thread handles it.